ilc/client.js

```diff
@@ -95,9 +95,14 @@ document.addEventListener('click', function (e) {
         return;
     }
 
-    const href = e.target.getAttribute('href');
-    singleSpa.navigateToUrl(href);
-    e.preventDefault();
+    const pathname = e.target.getAttribute('href');
+    const {routeId} = router.match(pathname);
+    const isSpecialRoute = Object.values(registryConf.specialRoutes).some((route) => route.routeId === routeId);
```

Comment:
Move `isSpecialRoute` attr to result of the `router.match()`

Reply:
Moved